microsoft-edge/extensions-chromium/publish/contact-us.md

```diff
@@ -0,0 +1,33 @@
+---
+description: Contact support information for Microsoft Edge extension development.
+title: Contact Edge extensions support
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 01/19/2021
+ms.topic: support
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, support
+---
+
+# Contact Edge extension support
+
+## Contact us for any issues or feedback
+
+1. For any issues related to certification, please submit a [New Support Request][ExtensionsSupportForm].
+1. If you have any issues or feedback regarding your publishing experience using Partner Center, write to us at [ext_dev_support@microsoft.com](mailto:ext_dev_support@microsoft.com).
```

Comment:
Suggest we make a small change here and first redirect to our publishing guide. 

If you are facing any issues while submitting your extension to the Partner Center review the tutorial to Publish your extensions. If that does not answer your question or you have a feedback regarding your publishing experience write to us at...